docs/resources/ocean_gke_launch_spec.md

```diff
@@ -142,6 +155,15 @@ The following arguments are supported:
     * `cpu_per_unit` - (Optional) Optionally configure the number of CPUs to allocate for each headroom unit. CPUs are denoted in millicores, where 1000 millicores = 1 vCPU.
     * `gpu_per_unit` - (Optional) Optionally configure the number of GPUS to allocate for each headroom unit.
     * `memory_per_unit` - (Optional) Optionally configure the amount of memory (MiB) to allocate for each headroom unit.
+* `network_interface` - (Optional) Settings for network interfaces.
+  * `network` - (Required) The name of the network.
+  * `project_id` - (Optional) Use a network resource from a different project. Set the project identifier to use its network resource. This parameter is relevant only if the network resource is in a different project.
+  * `access_configs` - (Optional) The network protocol of the VNG.
+    * `access_configs_name` - (Optional) The name of the access configuration.
```

Comment:
does not match with actual variable name used in TF, change it accordingly

Reply:
done